### DISCUSSION TITLE
posts/geultto-10th-resolution

### DISCUSSION THREAD

**Discussion by giscus** [witch-factory/witch-next-blog#46] [General]
# posts/geultto-10th-resolution

글또 10기를 시작하면서 쓰는 글

https://witch.work/posts/geultto-10th-resolution

**Comment by sewonkimm** [comment]
9기때 성현님과 만났던 시간은 저에게도 너무 좋은 시간이었습니다! 
9기 활동했을 때, 다양하게 많이 활동하신 것도 인상적이었어요. 그렇게 직접 부딫혀보셨으니 마음의 여유를 가져야한다는 결론을 내시게 된 거라고 생각해요. 느긋하게 쉬는건 저도 몇년간 많이 연습했는데... '일주일에 n시간 쉬기', '여행 n번 가기' 처럼 KPI, OKR처럼 수치화하는 순간 또 미션이 되더라고요..ㅠ
 
글을 읽다보니 깊이 몰입하는 것을 좋아하시는 성현님 성향을 고치지 않고, 자연스럽게 대입해서  현재에 깊이 몰입해보는 것도 여유를 찾는데 도움이 되지 않을까 싶네요! 좋은 감정이든, 나쁜 감정이든 찐~하게 소화해보시고... 여유로운 성현님이 되길 응원하겠습니다! 10기 같이 화이팅해요~👏

**Reply by sewonkimm** [reply]
오잉 습관성 물결표 사용으로 취소선이 생겼군요 XD
